modules/zero-trust-manager-pause-reconciliation.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * security/zero_trust_workload_identity_manager/zero-trust-manager-reconciliation.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="zero-trust-manager-pause-reconciliation_{context}"]
+
+= Pausing reconciliation by annotation
+
```

Comment:
Adding more explanation about the use case here, you could refer to: https://github.com/anirudhAgniRedhat/self-reference-doc/blob/b3ba2baf5bfa0c4fb34ad1a9fa7e723849334f84/ztwim/create-only-overide.md?plain=1#L11-L13